build.gradle

```diff
@@ -91,510 +104,48 @@ dependencies {
 	decompileClasspath "net.fabricmc:cfr:${project.cfr_version}"
 	mappingPoetJar "org.quiltmc:mapping-poet:${project.mapping_poet_version}"
 	unpick "net.fabricmc.unpick:unpick-cli:${project.unpick_version}"
+	hashed "org.quiltmc:hashed-mojmap:${minecraft_version}${USE_SNAPSHOT_HASHES ? "-SNAPSHOT" : ""}"
 }
 
-def setupGroup = "jar setup"
-def mappingsGroup = "quilt-mappings"
-def buildMappingGroup = "mapping build"
 def mapJarGroup = "jar mapping"
-def mappingsDir = file("mappings")
 def cacheFilesMinecraft = file(".gradle/minecraft")
 def tempDir = file(".gradle/temp")
 def mergedFile = file("${minecraft_version}-merged.jar")
-def hashedMojmapJar = file("${minecraft_version}-hashed-mojmap.jar")
-def unpickedJar = file("${minecraft_version}-hashed-mojmap-unpicked.jar")
+def hashedMojmapJar = file("${minecraft_version}-hashed.jar")
+def unpickedJar = file("${minecraft_version}-hashed-unpicked.jar")
 def namedJar = file("${minecraft_version}-named.jar")
-def versionFile = new File(cacheFilesMinecraft, "${minecraft_version}.json")
-def clientJar = new File(cacheFilesMinecraft, "${minecraft_version}-client.jar")
-def serverJar = new File(cacheFilesMinecraft, "${minecraft_version}-server.jar")
 def libraries = new File(cacheFilesMinecraft, "libraries")
```

Comment:
These can be changed by their `FileConstants` equivalent (you can get the instance with `project.getExtensions().getByType(MappingsExtension.class).getFileConstants()`)